BDArmory/Control/BDGenericAIBase.cs

```diff
@@ -247,6 +247,8 @@ public void TogglePilot()
         public virtual string Name { get; } = "AI Control";
         public bool Enabled => pilotEnabled;
 
+        public bool TakingOff = true;
```

Comment:
This is the same as `initialTakeOff` in BDModulePilot.cs. I'd suggest replacing `initialTakeOff` with this new `TakingOff` variable.
If we ever manage to make a landing routine, then that would need to reset this to `true`.